home/management/commands/updatecourses.py

```diff
@@ -0,0 +1,104 @@
+import requests
+from datetime import datetime
+
+from django.core.management import BaseCommand
+from django.db.models import Q
+
+from home.models import Course, Professor, ProfessorCourse
+from home.utils import Semester
+
+class Command(BaseCommand):
+    def __init__(self):
+        super().__init__()
+        self.total_num_new_courses = 0
+        self.total_num_new_professors = 0
+
+    def add_arguments(self, parser):
+        parser.add_argument("semesters", nargs='+')
+
+    def handle(self, *args, **options):
+        t_start = datetime.now()
+        inputted_semesters = [Semester(s).name() for s in options['semesters']]
+        print(f"Inputted Semesters: {', '.join(inputted_semesters)}")
+
+        for semester in options['semesters']:
+            s = Semester(semester)
```

Comment:
surely this is better written as 

```python
semesters = [Semester(s) for s in options['semesters']]
print(f"Inputted Semesters: {', '.join(s.name for s in semesters)}")

for semester in semesters:
    ...
```

than turning strings to semesters twice